app/src/main/res/layout/activity_custom_selector.xml

```diff
@@ -14,6 +15,25 @@
       android:layout_width="match_parent"
       android:layout_height="0dp"
       app:layout_constraintBottom_toBottomOf="parent"
-      app:layout_constraintTop_toBottomOf="@+id/toolbar_layout"/>
+      tools:layout_editor_absoluteX="16dp" />
+
+    <LinearLayout
+      android:id="@+id/linear_layout_noimage"
+      android:layout_width="204dp"
```

Comment:
Do not hardcode the values. As android screens have varied dimensions it might not look perfect on some screen sizes. Instead you can use `match_parent` or `wrap_content` that way the layout adjusts depending on the screen size.